### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by logrusx** [uBlockOrigin/uBOL-home#93] [General]
UBOLite worked better before the new versioning scheme. I now regret I updated it from 0.1.something (I have no way to find out what was the exact version).

It was able to block all ads from Facebook and only suggested posts could pull through. It was also being able to block some other marketing crap out from other regional sites.

I tried to find the tags or releases and I was disappointed not to find any.

Also the UI of some of the newer 2023 versions appears empty, but I guess this should be a separate issue.

**Comment by MasterKia** [comment]
> block some other marketing crap out from other regional sites.

We can't guess what those other sites are. Explain better.

**Comment by gorhill** [comment]
Moving to discussion as there is nothing specific and actionable.

**Comment by logrusx** [comment]
I apologize for not creating a discussion first place, this is actually the first time I see this GitHub feature.

What should I do about the empty UI? It might not be directly related to the extension although 2023.8.25.959 shows UI for me and the newer versions do not. In general that's not a problem, but sometimes I want to lower the level of filtering. Should I just create a new issue listing the software I'm using?

> > block some other marketing crap out from other regional sites.
> 
> We can't guess what those other sites are. Explain better.

The regional sites would be unusable as they are in a language I'm pretty sure no one here speaks(reads).

> Moving to discussion as there is nothing specific and actionable.

I'm not sure what would be actionable. I just see that FB ads escape the filter when there were literally non before the update and also some of the marketing tricks of the aforementioned regional sites. In the same time I have no clue about how this extension works and extensions in general so I have no idea what would be relevant.
